src/EnergyPlus/EconomicTariff.cc

```diff
@@ -4581,9 +4640,14 @@ namespace EconomicTariff {
 		numMins = 0;
 		MinTariffIndex.dimension( numTariff, 0 );
 		for ( iTariff = 1; iTariff <= numTariff; ++iTariff ) {
-			//determine if this is meter related to electricity
+			// TODO: is this needed?
+			// Assume it's not a water meter (already handled in the constructor, since set to 0)
+			tariff( iTariff ).kindWaterMtr = kindMeterNotWater;
+
+			//determine if this is meter related to electricity, or water
 			if ( tariff( iTariff ).reportMeterIndx != 0 ) {
```

Comment:
This is more a question for @JasonGlazer .  If the kind of meter has already been set in GetInputEconomicsTariff, why does this logic need to be repeated here?

Reply:
Can I get a ruling on this please? I think it makes more sense to move it in `GetInputEconomicsTariff`